bika/lims/content/analysisrequest.py

```diff
@@ -2551,6 +2551,12 @@ def getSamplingRoundUID(self):
         else:
             return ''
 
+    def getStorageLocationTitle(self):
+        """ A method for AR listing catalog metadata
+        :return: Title of Storage Location
+        """
+        return self.getStorageLocation() and self.getStorageLocation().Title() or ''
```

Comment:
You are calling here a "potentailly expensive" method twice. Better do it once and keep it in a variable